src/_P087_SerialProxy.ino

```diff
@@ -238,6 +240,49 @@ boolean Plugin_087(uint8_t function, struct EventStruct *event, String& string)
         }
       }
 
+      if (cmd.equalsIgnoreCase(F("serialproxy_write_tr"))) {
```

Comment:
`cmd` is already lowercase, (`parseString()` does that) so no need to do a case-insensitive compare, that could save a few CPU cycles...